TTS/vocoder/configs/hifigan_config.py

```diff
@@ -104,6 +104,7 @@ class HifiganConfig(BaseGANVocoderConfig):
             "resblock_type": "1",
         }
     )
+    discriminator_model_params: dict = field(default_factory=lambda: {"periods": [2, 3, 5, 7, 11]})
```

Comment:
HifiGAN changes must be a separate PR

Reply:
Yeah, my bad it is in the PR: https://github.com/coqui-ai/TTS/pull/1526